src/visualization/types/SimpleTable/options.tsx

```diff
@@ -0,0 +1,6 @@
+import React, {FC} from 'react'
+const SimpleTableOption: FC = () => {
+  return <h5>There is nothing to configure here</h5>
+}
+
+export default SimpleTableOption
```

Comment:
question - did SimpleTable get bounced to giraffe yet?

Reply:
not yet. working on it